### PR TITLE
Complete Appartment Simulation

### PR DESCRIPTION
**Pull request by DavidMansolino**
Preview:
complete_apartment

https://cyberbotics.com/doc/guide/samples-environments?version=feature-appartment#complete_apartment-wbt

### CODE REVIEW

**Comment by DavidMansolino** [comment]
New version:
complete_apartment_2

**Comment by DavidMansolino** [comment]
:+1: 

a good overview is visible at https://cyberbotics.com/doc/guide/samples-environments?version=feature-appartment#complete_apartment-wbt

**Comment by fabienrohrer** [comment]
👀

File: docs/guide/samples-environments.md [hunk 1]
```diff
@@ -28,6 +28,23 @@ This means that a robot could open a drawer, leave an object inside it, and clos
 Light objects such as the books on the shelf or the fruits in the basket can be grabbed by a robot.
 An e-puck robot patrols on the table and an iRobot Create robot cleans the ground.
 
+### [complete\_apartment.wbt](https://github.com/omichel/webots/tree/master/projects/samples/environments/indoor/worlds/complete_apartment.wbt)
+
+**Keywords**: Apartment, domestic environment, furniture
```
Comment (fabienrohrer) [inline docs/guide/samples-environments.md]: `domestic` has connotations and `domestic environment` is a business term. What about `household` instead?

```suggestion
**Keywords**: Apartment, household environment, furniture
```

**Comment by DavidMansolino** [comment]
:+1:

File: docs/guide/samples-environments.md [hunk 2]
```diff
@@ -28,6 +28,23 @@ This means that a robot could open a drawer, leave an object inside it, and clos
...
+|     |     |
+|:---:|:---:|
+| ![complete_apartment_a.png](images/samples/complete_apartment_a.thumbnail.jpg) | ![complete_apartment_b.png](images/samples/complete_apartment_b.thumbnail.jpg) |
+| ![complete_apartment_c.png](images/samples/complete_apartment_c.thumbnail.jpg) | ![complete_apartment_d.png](images/samples/complete_apartment_d.thumbnail.jpg) |
+
+%end
+
+This example is similar to the previous one but the apartement is more complete, it also contains two bedrooms, one bathroom, one restroom, stairs and a balcony.
+
+> **Note**: This simulation was sponsored by Kamal Othman & Prof. Ahmad Rad from AISL-SFU
```
Comment (omichel) [inline docs/guide/samples-environments.md]: I would say "this model" instead of "this simulation"

**Pull request merged**